### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by MehmetSunguralp**
image

### CODE REVIEW

File: cta/cta.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+body {
+	margin: 0;
+	font-family: 'Inter', sans-serif;
+}
+
+#cta-base {
```
Comment (kumbu132) [inline cta/cta.css]: - [ ] Lets try to avoid using ids in our CSS. lets stick with classes only

File: cta/cta.css [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	background: #2ec5ce;
+	height: 491px;
```
Comment (kumbu132) [inline cta/cta.css]: - [ ] Use rounded/even values for px

File: cta/cta.css [hunk 3]
```diff
@@ -0,0 +1,47 @@
+body {
+	margin: 0;
```
Comment (kumbu132) [inline cta/cta.css]: - [ ] define resets like `margin: 0` in a global css file. (i.e. 
```
* {
        padding: 0;
        margin: 0;
        box-sizing: border-box;
      }
```

File: cta/cta.css [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+#cta-text {
+	font-size: 18px;
+	line-height: 32px;
+}
+
+#cta-sub-title {
+	font-size: 18px;
+	font-weight: 500;
+	line-height: 28px;
+	margin: 32px 0 10px 0;
```
Comment (kumbu132) [inline cta/cta.css]: - [ ] - Simplify this to `margin: 32px 0 10px;`

File: cta/cta.css [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.cta-badges {
+	margin: 0 6px 0 0;
```
Comment (kumbu132) [inline cta/cta.css]: - [ ] Make this margin-right: 6px;

File: cta/cta.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```
Comment (kumbu132) [inline cta/cta.html]: The indentation here could be better to make it more readable. Here's how to set up your Visual Stuido Code to make it better in terms of readability. 

1. Open VS Code
2. Search for `Prettier` in extensions and install it.. 
image

3.  Open settings (you can use the keyboard shortcut `Ctrl + ,`)
4. Search for `formatter` and make sure prettier is enabled as the default formatter
image

5. Make sure that `Format on Paste` and `Format on Save` are checked. 
image

6. Save all your files again (i.e. open them and press `Ctrl + S`) and watch them get formatted properly.

**Review by kumbu132 — CHANGES_REQUESTED** [review]
Good job so far. I tested it out, and it looks good on my phone too. The coding, however, could be improved. Some of the comments I made apply to your other PRs as well, let's update them accordingly and then we can merge. Great work.

File: cta/cta.css (hunk 2 above)
Comment (MehmetSunguralp) [reply cta/cta.css]: I think, I directly copied the value from the original design. I'll fix it, though.

**Pull request merged**